cmd/sealer/cmd/cluster/run.go

```diff
@@ -253,9 +253,10 @@ func createNewCluster(imageEngine imageengine.Interface, cf clusterfile.Interfac
 	}
 
 	runtimeConfig := &clusterruntime.RuntimeConfig{
-		Distributor: distributor,
-		ImageEngine: imageEngine,
-		Plugins:     plugins,
+		Distributor:            distributor,
+		ImageEngine:            imageEngine,
+		Plugins:                plugins,
+		ContainerRuntimeConfig: cluster.Spec.ContainerRuntime,
```

Comment:
could we get ContainerRuntime from image spec(image extension) as default value, using cluster.Spec.ContainerRuntime  to ovewrite it ?

Reply:
i think after we add ContainerRuntime info in  image spec, then modify here.